app/views/WishList/index.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Button, Container, ElementFragments, InformationCard, NumberInput, TextOutput } from '@the-deep/deep-ui';
+import {
+    gql,
+    useQuery,
+    useMutation,
+} from '@apollo/client';
+import { AiTwotoneDelete } from 'react-icons/ai';
+import { FaRegHeart, FaShoppingCart } from 'react-icons/fa';
+
+import {
+    WishListQuery,
+    WishListQueryVariables,
+} from '#generated/types';
+
+import styles from './styles.css';
+
+const WISH_LIST = gql`
+    query WishList ($pageSize: Int!, $page: Int!){
+        wishList(pageSize: $pageSize, page: $page) {
+            results {
+                book {
+                    id
+                    isbn
+                    authors {
+                        id
+                        name
+                    }
+                    price
+                    title
+                    language
+                    image {
+                        url
```

Comment:
If there's an `id` we should always fetch `id`

Reply:
There is no id for image in API